install.sh

```diff
@@ -0,0 +1,454 @@
+#!/bin/sh
+set -e
+
+
+# The install script is licensed under BSD 2-Clause "Simplified" License.
+
+usage() {
+  this=$1
+
+  cat <<EOF
+Download DeepSource CLI binary from https://github.com/deepsourcelabs/cli
+EOF
+  exit 2
+}
+
+parse_args() {
+  #BINDIR is ./bin unless set be ENV
+  # over-ridden by flag below
+
+  BINDIR=${BINDIR:-./bin}
+  while getopts "b:dh?x" arg; do
+    case "$arg" in
+      b) BINDIR="$OPTARG" ;;
+      d) log_set_priority 10 ;;
+      h | \?) usage "$0" ;;
+      x) set -x ;;
+    esac
+  done
+  shift $((OPTIND - 1))
+  TAG=$1
+
+}
+# this function wraps all the destructive operations
+# if a curl|bash cuts off the end of the script due to
+# network, either nothing will happen or will syntax error
+# out preventing half-done work
+execute() {
+  tmpdir=$(mktemp -d)
+  log_debug "downloading files into ${tmpdir}"
+  http_download "${tmpdir}/${TARBALL}" "${TARBALL_URL}"
+  http_download "${tmpdir}/${CHECKSUM}" "${CHECKSUM_URL}"
+  hash_sha256_verify "${tmpdir}/${TARBALL}" "${tmpdir}/${CHECKSUM}"
+  srcdir="${tmpdir}"
+  (cd "${tmpdir}" && untar "${TARBALL}")
+  test ! -d "${BINDIR}" && install -d "${BINDIR}"
+  for binexe in $BINARIES; do
+    if [ "$OS" = "windows" ]; then
+      binexe="${binexe}.exe"
+    fi
+    install "${srcdir}/${binexe}" "${BINDIR}/"
+    log_info "installed ${BINDIR}/${binexe}"
+  done
+  rm -rf "${tmpdir}"
+}
+get_binaries() {
+  case "$PLATFORM" in
+    darwin/386) BINARIES="deepsource" ;;
+    darwin/amd64) BINARIES="deepsource" ;;
+    darwin/arm64) BINARIES="deepsource" ;;
+    darwin/armv6) BINARIES="deepsource" ;;
+    freebsd/386) BINARIES="deepsource" ;;
+    freebsd/amd64) BINARIES="deepsource" ;;
+    freebsd/arm64) BINARIES="deepsource" ;;
+    freebsd/armv6) BINARIES="deepsource" ;;
+    linux/386) BINARIES="deepsource" ;;
+    linux/amd64) BINARIES="deepsource" ;;
+    linux/arm64) BINARIES="deepsource" ;;
+    linux/armv6) BINARIES="deepsource" ;;
+    netbsd/386) BINARIES="deepsource" ;;
+    netbsd/amd64) BINARIES="deepsource" ;;
+    netbsd/arm64) BINARIES="deepsource" ;;
+    netbsd/armv6) BINARIES="deepsource" ;;
+    openbsd/386) BINARIES="deepsource" ;;
+    openbsd/amd64) BINARIES="deepsource" ;;
+    openbsd/arm64) BINARIES="deepsource" ;;
+    openbsd/armv6) BINARIES="deepsource" ;;
+    windows/amd64) BINARIES="deepsource" ;;
+    *)
+      log_crit "platform $PLATFORM is not supported.  Make sure this script is up-to-date and file request at https://github.com/${PREFIX}/issues/new"
+      exit 1
+      ;;
+  esac
```

Comment:
Doesn't work on Windows for some reason. Tested on Cygwin/Git Bash environment.

Install script doesn't work with Powershell too.

Reply:
I can't find any other change that needs to be made specifically for windows 🤔